### ISSUE TITLE
Typings issue faced when compiling using ViteJS

### ISSUE THREAD

**Issue by SyedUmerHasan** [numldesign/tasty#1]
**Is your feature request related to a problem? Please describe.**
```
import React from 'react';
import { Base } from 'tastycss/dist/mjs/react';
import { AllBaseProps } from 'tastycss/dist/mjs/types/component';

const BaseElement = React.forwardRef((props: AllBaseProps, ref) => (<Base {...props} ref={ref}>
  {props.children}
</Base>));

export default BaseElement;
```
Yes, While I am building TastyCSS. ViteJs gives
```
[vite:resolve] Missing "./dist/mjs/react" export in "tastycss" package
error during build:
Error: Missing "./dist/mjs/react" export in "tastycss" package
```
This means in tastycss package.json, **tastycss/dist/mjs/react** entry in exports is not found. 

**Describe the solution you'd like**
Need to see, what will be the actual way to import modules from TastyCSS

**Describe alternatives you've considered**

**Additional context**

**Comment by SyedUmerHasan** [comment]
After seeing, 
https://github.com/numldesign/tasty/blob/8589bd8e40b31df877c37f971abc6403ddf2a0ca/package.json#L25-L29
I have tried this. 

```
import React from 'react';
import { Base } from 'tastycss/react';
import { AllBaseProps } from 'tastycss/dist/mjs/types/component';

const BaseElement = React.forwardRef((props: AllBaseProps, ref) => (<Base {...props} ref={ref}>
  {props.children}
</Base>));

export default BaseElement;
```
Project is bundling but shows typings are not found 
```
BaseElement/src/BaseElement.tsx:2:22 - error TS2307: Cannot find module 'tastycss/react' or its corresponding type declarations.

2 import { Base } from 'tastycss/react';
                       ~~~~~~~~~~~~~~~~
Found 1 error.
```

**Comment by SyedUmerHasan** [comment]
Another thing I would like to add @tenphi
When I use 
```
import React from 'react';
const Tasty = require('tastycss/react');
import { AllBaseProps } from 'tastycss/dist/mjs/types/component';

const BaseElement = React.forwardRef((props: AllBaseProps, ref) => (<Tasty.Base {...props} ref={ref}>
  {props.children}
</Tasty.Base>));

export default BaseElement;
```
It works perfectly.

**Comment by tenphi** [comment]
The correct way to use it is:

```
import React from 'react';
import { Base, AllBaseProps } from 'tastycss/react';
```

There were some issues with typings but I fixed it in version `0.0.7`.
